### ISSUE TITLE
Display Issues

### ISSUE THREAD

**Issue by float3** [git-truck/git-truck#605]
https://user-images.githubusercontent.com/86748455/200020851-b2cf05a3-cccc-483b-96f1-7016e5c9562f.mp4

![image](https://user-images.githubusercontent.com/86748455/200021025-4841377b-16cc-4fe3-8f78-c0fa6b27bca9.png)
![image](https://user-images.githubusercontent.com/86748455/200021078-85e2acb0-5462-4fff-b87d-c2e4f50823c7.png)

I've tried multiple browsers, including:
Librewolf
Firefox
Chrome
Opera

git truck looks the same on all of them; I am on the latest version by `npx git-truck@latest.`

**Comment by joglr** [comment]
Can you provide some more details? What is your OS, Node version, what is printed in the console?
 (both browser console and the terminal you started git-truck in)

**Comment by float3** [comment]
`npx --version` is 8.5.5
the issues only happen on windows 10 (I also tested on archlinux)
Console output:
```
Git Truck version 1.2.5 (latest)

✔ Git Truck started [503ms]
✔ Done checking for git repo [243ms]
✔ Succesfully opened Git Truck in your browser [5ms]

Application available at http://localhost:3000
✔ Found branch head [24ms]
GET /vrc-udon-audio-link/dev 200 - - 1113.485 ms
GET /favicon.ico 302 - - 0.969 ms
✔ Done checking if path is a git repo [52ms]
✔ Done scanning for repositories [159ms]
GET / 200 - - 250.863 ms
```
![image](https://user-images.githubusercontent.com/86748455/200147909-677299d8-c36d-403c-bce5-294aa430d4c3.png)

**Comment by float3** [comment]
@joglr this is solved now (on git-truck 1.3.0)

**Comment by joglr** [comment]
> @joglr this is solved now (on git-truck 1.3.0)

That's good to hear!
